src/API/Reports/Categories/DataStore.php

```diff
@@ -114,7 +114,16 @@ protected function get_sql_query_params( $query_args ) {
 			$sql_query_params['where_clause'] .= " AND ( {$order_status_filter} )";
 		}
 
-		$sql_query_params['where_clause'] .= " AND taxonomy = 'product_cat' ";
+		$where_clause = $sql_query_params['where_clause'] . " AND taxonomy = 'product_cat' ";
+		/**
+		 * Filter the notes WHERE clause before retrieving the category data.
+		 *
+		 * Allows modification of the notes select criteria.
+		 *
+		 * @param string $where_clause The generated WHERE clause.
+		 * @param array  $query_args   The original arguments for the request.
+		 */
+		$sql_query_params['where_clause'] = apply_filters( 'wc_admin_categories_where_clause', $where_clause, $query_args );
```

Comment:
Should this include `$order_product_lookup_table` like the call to `get_time_period_sql_params()`?